client/controller/api/apps.go

```diff
@@ -6,7 +6,7 @@ type App struct {
 	ID      string `json:"id"`
 	Owner   string `json:"owner"`
 	Updated string `json:"updated"`
-	URL     string `json:"url"`
+	URL     string
```

Comment:
Also, see [this code sample](http://play.golang.org/p/Qzfal52SJD) for an illustration of the case when `"URL"` is present and not present in the incoming JSON